claim/README.md

```diff
@@ -83,21 +97,38 @@ can trust the endpoint if necessary.
 Claiming can be performed through a SOCKS5 proxy. To do this when calling the script directly supply the proxy
```

Comment:
We can also use http proxies

Reply:
Additionally for SOCKS5 support `curl` must be present in system as `wget` doesn't support SOCKS5 unfortunatelly. This is however handled in claiming script with error message. (it is more of an issue if claiming from agent as that message then goes to error.log and is not instantly visible)